.github/workflows/_test_pax_rosetta.yaml

```diff
@@ -471,17 +476,17 @@ jobs:
     if: ( always() )
     secrets: inherit
     with:
-      ENDPOINT_FILENAME: 'pax-test-status.json'
+      ENDPOINT_FILENAME: '${{ inputs.ARTIFACT_NAME }}pax-test-status.json'
```

Comment:
We should probably change this since the default behavior is for the endpoints in this file and in _test_pax.yaml to collide
```suggestion
      ENDPOINT_FILENAME: '${{ inputs.ARTIFACT_NAME }}rosetta-pax-test-status.json'
```

Reply:
We could make `inputs.ARTIFACT_NAME` required as well to ensure people try to set this, but if someone enters `""`, they'd still end up with endpoints that collide

Reply:
![image](https://github.com/NVIDIA/JAX-Toolbox/assets/6081524/dd055ee4-d958-4a6c-8f04-c67a2001cb81)

@terrykong Did you make this suggestion or did some copilot bot do it?

Reply:
![Screenshot 2023-10-17 at 3 57 18 PM](https://github.com/NVIDIA/JAX-Toolbox/assets/7576060/f0d8fcd8-eb5e-49b2-b434-23800b981fe1)

Actually it's supported by github; just gotta click this button. You can also do multiline suggestions as well